.github/workflows/storybook.yml

```diff
@@ -43,4 +43,3 @@ jobs:
                   token: ${{ secrets.GITHUB_TOKEN }}
                   projectToken: ${{ secrets.CHROMATIC_APP_CODE }}
                   autoAcceptChanges: "main"
-                  onlyStoryFiles: "**/*regression.stories.tsx"
```

Comment:
So I think the issue is that `onlyStoryFiles` is an option inside the `chromatic.config.json` file, the option is here: https://www.chromatic.com/docs/configure/#options

Reply:
Ah, that's good to know!

I think we're gonna enable a few key stories other than the regression stories soon, so I think it would still be worth removing this from the .yml file.

Reply:
It seems like this is working.  This setting is documented as being supported in the GitHub Action: 

<img width="815" alt="image" src="https://github.com/user-attachments/assets/43403dba-a76d-4f17-9a8c-20ad9d1b4d9a">

On an [example PR](https://github.com/Khan/perseus/actions/runs/12017642792/job/33500417832?pr=1906) I checked, it seems like it is obeying the setting: 

```
19:50:09.030 Starting partial build
19:50:09.030     → Snapshots will be limited to story files matching '**/*regression.stories.tsx'

...

19:50:58.649 Build 5134 passed!
                 → Tested 412 stories across 130 components; captured 27 snapshots in 39 seconds
```

It captured only 27 snapshots.

Reply:
@jeremywiebe It does say "captured 27 snapshots" on all the PRs. I checked a bunch of older unrelated Perseus PRs to check this, and it's true that that messaging is consistent. However, the PRs would _also_ show other snapshots that aren't part of these 27 in the "UI test" check. It also says there are 130 tests on the chromatic dashboard for other PRs when there should only be 27:

<img width="213" alt="image" src="https://github.com/user-attachments/assets/c31f8d37-cda8-4e86-91cd-76a9569ccc5e">


I'd like to move away from using the `onlyStoryFiles: "**/*regression.stories.tsx"` because I think there's a disconnect somewhere that isn't super obvious.

Reply:
I'm going to retract my last comment. Just for a confidence check, I put up another PR that would have had a ton of UI test updates if my assertion was correct. It only reported changes in the regression stories, so I think my statements before this were all false, and I just don't know what "tests" means on the chromatic dashboard.

Reply:
@jeremywiebe we can still continue with this PR if we want to enable other stories to have snapshots later, rather than just limiting to regression stories.